Phase05/SampleLibrary/SampleLibrary.Test/InvertedIndexTest.cs

```diff
@@ -0,0 +1,51 @@
+using FluentAssertions;
+
+namespace SampleLibrary.Test;
+
+public class InvertedIndexTest
+{
+    [Fact]
+    public void InvertedIndex_InsertSomeWords_ReturnsTrue()
+    {
+        var invertedIndex = new InvertedIndex();
+
+        invertedIndex.indexDocument("hello ali", "1");
+        invertedIndex.indexDocument("this is a code", "2");
+        invertedIndex.indexDocument("to search some words", "3");
+        invertedIndex.indexDocument("in a text, ali", "4");
+
+        var map = invertedIndex.Map;
+
+        map.ContainsKey("hello").Should().Be(true);
+        map.ContainsKey("mohammad").Should().NotBe(true);
+
+        map["ali"].Contains("1").Should().Be(true);
+        map["ali"].Contains("4").Should().Be(true);
+
+        map["ali"].Contains("2").Should().Be(false);
+    }
+
+    [Fact]
+    public void InvertedIndexMaker_InsetSomeWordsInInvertedIndex_ReturnsTrue()
+    {
```

Comment:
confusing names
what is the difference between this test and the one before it?
And why does the name end with ReturnTrue when you are asserting certain values not to be true at the end?